rabbitmq/tests/metrics.py

```diff
@@ -37,6 +37,15 @@
         ]
     )
 
+# Only present in 3.8 or greater
+if RABBITMQ_VERSION >= version.parse('3.8'):
+    E_METRICS.extend(
+        [
+            'rabbitmq.exchange.messages.drop_unroutable.count',
+            'rabbitmq.exchange.messages.drop_unroutable.rate',
+        ]
+    )
+
```

Comment:
Would be nice to add a new env with this version to test it 🙂

Reply:
Here: https://github.com/DataDog/integrations-core/blob/master/rabbitmq/hatch.toml#L10

Reply:
I  figured out that the drop metric is provided only in the overview API and not in the exchange API.
So I removed the wrong code.